projects/packages/jetpack-mu-wpcom/src/features/wpcom-options-general/options-general.ts

```diff
@@ -55,10 +55,28 @@ const _wpcomBuildSiteUrl = ( fragment: DocumentFragment ) => {
 		const homeUrlLabel = __( 'Site Address (URL)', 'jetpack-mu-wpcom' );
 		const homeUrlRow = document.createElement( 'tr' );
 		homeUrlRow.innerHTML = `
-					<th scope="row"><label for="home">${ homeUrlLabel }</label></th>
-					<td><input type="url" id="home" value="${ window.wpcomSiteUrl.homeUrl }" class="regular-text code disabled" disabled="disabled" /></td>
-			`;
+			<th scope="row"><label for="home">${ homeUrlLabel }</label></th>
+			<td>
+				<input type="url" id="home" value="${ window.wpcomSiteUrl.homeUrl }" class="regular-text code disabled" disabled="disabled" />
+				<button class="button wpcom-add-custom-address-button">
```

Comment:
We should change this line to prevent the same issue as https://github.com/Automattic/jetpack/pull/42093/ :D

```suggestion
				<button type="button" class="button wpcom-add-custom-address-button">
```